amd_openvx_extensions/amd_rpp/CMakeLists.txt

```diff
@@ -176,6 +176,7 @@ list(APPEND SOURCES
         source/tensor/WarpAffine.cpp
         source/tensor/Slice.cpp
         source/tensor/Transpose.cpp
+        source/tensor/TensorLog1p.cpp
```

Comment:
Why is it called TensorLog1p?

Reply:
For all arithmetic ops, they have Tensor as a prefix like TensorAdd, TensorSubtract. Its to indicate that they are doing the operation on the tensor